sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/ResolveGroupingAnalyticsSuite.scala

```diff
@@ -41,61 +41,64 @@ class ResolveGroupingAnalyticsSuite extends AnalysisTest {
   lazy val r1 = LocalRelation(a, b, c)
 
   test("rollupExprs") {
-    val testRollup = (exprs: Seq[Expression], rollup: Seq[Seq[Expression]]) => {
-      val result = SimpleAnalyzer.ResolveGroupingAnalytics.rollupExprs(exprs)
+    val testRollup = (exprs: Seq[Seq[Expression]], rollup: Seq[Seq[Expression]]) => {
+      val result = GroupingSet.rollupExprs(exprs)
```

Comment:
the input can still be `Seq[Expression]`, and here we do `GroupingSet.rollupExprs(exprs.map(Seq(_)))`

Reply:
> the input can still be `Seq[Expression]`, and here we do `GroupingSet.rollupExprs(exprs.map(Seq(_)))`

Done